### ISSUE TITLE
"Error getting the response."

### ISSUE THREAD

**Issue by woody4165** [vukmirovic98/node-red-contrib-covid19#3]
Hi

Is there any issue with the node?

I'm getting this error if I try a single country as well as if I try several countries together

Thanks

**Comment by woody4165** [comment]
Hi @vukmirovic98 

I see that a new version, 2.0.1 is out that probably resolve the issue I've opened, Thanks!

Will I find it as update in the Manage Palette in NR or should I install via NPM? I ask you this because now is not available as update.

**Comment by JungeG** [comment]
Hi!
when calling the node for Germany, I receive following answer:

> 22.4.2020, 09:22:13node: COVID19 Deutschland
msg : string[27]
"Error getting the response."

The worldwide node is working fine.

I installed version 2.0.1 some error message.
Any idea?
Thanks Georg

**Comment by nygma2004** [comment]
Yes, same for me. I did not update for some time now, mine broke sometime last week. Everything was working up until that point.

**Comment by nygma2004** [comment]
> Hi! when calling the node for Germany, I receive following answer:
> 
> > 22.4.2020, 09:22:13node: COVID19 Deutschland
> > msg : string[27]
> > "Error getting the response."
> 
> The worldwide node is working fine.
> 
> I installed version 2.0.1 some error message. Any idea? Thanks Georg

I did some digging of the original library used by this node. You can simply use a HTTP request node and call the following URL: https://disease.sh/v3/covid-19/countries/DE?strict=true
It gives the same data as this node. You can use this until this node gets fixed.

**Comment by vukmirovic98** [comment]
Sorry, this was one of my first nodes, and honestly, the API provider made a lot of changes since deploying, but hadn't had time to make updates
